src/framework/card/Card.jsx

```diff
@@ -59,12 +59,15 @@ export default class Card extends Component {
             </div>
           </div>
         </div>
+
+        {this.props.children}
```

Comment:
Feels weird to be passed as `children`. Would have gone for `CardRibbon` and `ribbon` prop. It's somehow misplaced as a generic child.